### PR TITLE
Created Header and footer template

### PR DESCRIPTION
**Pull request by the-friyia**
Hello guys I made the header and footer templates for the website. Please test and +1

@woodenchalet @RaufHossain @ZacharyLau

### CODE REVIEW

**Comment by woodenchalet** [comment]
Could you add some comments/docstring  plz.

**Comment by the-friyia** [comment]
Finished those changes @woodenchalet I also separated out the CSS and JavaScript imports on the front end so that we do not have to memorize them. We just need to <%include import-js%> and the server will assemble the pages and send them to the client

**Comment by RaufHossain** [comment]
+1

File: views/index.ejs [hunk 1]
```diff
@@ -1,19 +1,20 @@
 <html>
```
Comment (woodenchalet) [inline views/index.ejs]: The docstring for this file pleased. Thank you.

**Comment by the-friyia** [comment]
@woodenchalet I accidentally pushed to master but the docstring is there in index.ejs. Sorry about that !!! I screwed up

**Comment by the-friyia** [comment]
I will make a new pull request if you would like some other changes

**Comment by woodenchalet** [comment]
It's ok. No worries.

**Comment by woodenchalet** [comment]
+1

**Pull request merged**